src/app/services/notify.service.ts

```diff
@@ -18,10 +18,15 @@ export class NotifyService {
     });
   }
 
-  public showMessage(message: string, type: string): void {
+  public showMessage(message: string, messageType: string): void {
```

Comment:
if any thing called like ...blablaType -> create for it separate type. TypeScript - it is about type